### PR TITLE
Add pre-commit linters

### PR DESCRIPTION
**Pull request by zaro0508**
Setup pre-commit yamllint and cfn-lint for syntax validation.

### CODE REVIEW

**Comment by jfalkenstein** [comment]
Let's hold off on merging this until I get my current PRs all merged in.

**Comment by jfalkenstein** [comment]
@zaro0508, we can proceed with this one now

File: sceptre-example/config/lambda-stack-bootstrapless.yaml
Comment (zaro0508) [inline sceptre-example/config/lambda-stack-bootstrapless.yaml:21]: quoted to fix the following error..

```
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

sceptre-example/config/lambda-stack-bootstrapless.yaml
  21:47     error    syntax error: expected <block end>, but found '<scalar>' (syntax)
```

Comment (jfalkenstein) [reply sceptre-example/config/lambda-stack-bootstrapless.yaml:21]: Thanks. There are several that started failing on the main branch once I activated webhooks. 🤦 I didn't realize it wasn't running tests until after I merged. But we're not quite ready to deploy for the first time anyway, so I think we're still ok. We're still in "alpha". I'll give you 1000 Jon points if you manage to get all the tests passing.

Comment (zaro0508) [reply sceptre-example/config/lambda-stack-bootstrapless.yaml:21]: Are Jon points redeemable for anything?  Like one of those big stuffed animal from the circus?

**Review by jfalkenstein — APPROVED** [review]

**Pull request merged**
